test.js

```diff
@@ -0,0 +1,154 @@
+const puppeteer = require('puppeteer');
+const pageUrl = "http://localhost:3101/";
+const db = require('./database/index.js');
+
+let page;
+let browser;
+const width = 1200;
+const height = 700;
+
+beforeAll(async () => {
+  // db.seed();
+
+  // browser = await puppeteer.launch({
+  //   headless: false,
+  //   slowMo: 80,
+  //   args: ['--window-size=${width}, ${height}']
+  // });
+  // page = await browser.newPage();
+  // await page.setViewport({ width, height });
+});
+afterAll(() => {
+  // browser.close();
+});
```

Comment:
Empty function invocations. Their internal code should be uncommented, and these should go into a `/** */` block.